submodules/k8s/templates/k8s-functions.sh.tftpl

```diff
@@ -32,6 +32,22 @@ set_k8s_auth() {
   echo
 }
 
+set_eniconfig() {
+  local ENICONFIG_YAML="${eniconfig_yaml}"
+  if [ -z "$ENICONFIG_YAML" ]; then
+      return
+  fi
+  kubectl_cmd -n kube-system set env daemonset aws-node AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG=true
+  if test -f "$ENICONFIG_YAML"; then
+    printf "$GREEN Updating $ENICONFIG_YAML... $EC \n"
+    kubectl_apply "$ENICONFIG_YAML"
+  else
+    printf "$RED $ENICONFIG_YAML does not exist. $EC \n" && exit 1
+  fi
+  echo
+  kubectl_cmd -n kube-system set env daemonset aws-node ENI_CONFIG_LABEL_DEF=topology.kubernetes.io/zone
```

Comment:
Just from a safety-ish perspective, why not apply the aws-node changes once since `set env` takes multiple env variables? Would it also be worth `kubectl wait`'ing to ensure it rolls out successfully and isn't left crashlooping?

Reply:
For the first question, just dogmatically doing things in the same order as the aws docs. There's probably no good reason for it. For the second, there's nothing to roll out since there are no nodes.

Reply:
Updated this to a single call.

Reply:
Oh yeah... duh. Thanks.